BrocoliSe/BrocoliSe/Utilities/TableViewExtension.swift

```diff
@@ -9,9 +9,9 @@ import Foundation
 import UIKit
 
 extension UITableView {
-    func setEmptyView(for type: EmptyViewType) {
+    func setEmptyView(for type: EmptyViewType, tapButton: UITapGestureRecognizer? = nil) {
```

Comment:
Acho q tem q dar um enter pra quebrar a linha
Function Body Length Violation: Function body should span 40 lines or less excluding comments and whitespace: currently spans 51 lines (function_body_length)